README.md

```diff
@@ -52,48 +54,52 @@ Then, in your `.eslintrc.json`:
 
 ## Recommended Configuration
 
-This plugin works best if you disable all other ESLint rules relating to code formatting, and only enable rules that detect patterns in the AST. (If another active ESLint rule disagrees with `prettier` about how code should be formatted, it will be impossible to avoid lint errors.) You can use [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) to disable all formatting-related ESLint rules.
-
-If your desired formatting does not match the `prettier` output, you should use a different tool such as [prettier-eslint](https://github.com/prettier/prettier-eslint) instead.
+This plugin works best if you disable all other ESLint rules relating to code formatting, and only enable rules that detect potential bugs. (If another active ESLint rule disagrees with `prettier` about how code should be formatted, it will be impossible to avoid lint errors.) You can use [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) to disable all formatting-related ESLint rules.
 
-To integrate this plugin with `eslint-config-prettier`, you can use the `"recommended"` configuration:
+This plugin ships with a `plugin:prettier/recommended` config that sets up both the plugin and `eslint-config-prettier` in one go.
 
 1. In addition to the above installation instructions, install `eslint-config-prettier`:
 
    ```sh
    npm install --save-dev eslint-config-prettier
    ```
 
-2. Then you need to add `plugin:prettier/recommended` as the last extension in your `.eslintrc.json`:
+   Compatibility:
+
+   - eslint-plugin-prettier@>=4 ↔ eslint-config-prettier@>=7
+   - eslint-plugin-prettier@<4 ↔ eslint-config-prettier@<7
+
+2. Then you need to add `plugin:prettier/recommended` as the _last_ extension in your `.eslintrc.json`:
 
    ```json
    {
      "extends": ["plugin:prettier/recommended"]
    }
    ```
 
-This does three things:
+   See [eslint-config-prettier’s explanation](https://github.com/prettier/eslint-config-prettier#eslint-plugin-prettier) for more information about what this config does.
```

Comment:
This link is broken at the moment but will work once https://github.com/prettier/eslint-config-prettier/pull/168 is merged.

Preview: https://github.com/prettier/eslint-config-prettier/tree/next#eslint-plugin-prettier